src/main/java/htsjdk/samtools/cram/io/DefaultBitOutputStream.java

```diff
@@ -196,21 +159,61 @@ public void flush() {
     }
 
     @Override
-    public void write(@SuppressWarnings("NullableProblems") final byte[] b) {
+    public void close() {
+        flush();
         try {
-            out.write(b);
+            out.close();
         } catch (final IOException e) {
             throw new RuntimeIOException(e);
         }
     }
 
-    @Override
-    public void write(@SuppressWarnings("NullableProblems") final byte[] b, final int off, final int length) {
+    public void write(final byte b) {
         try {
-            out.write(b, off, length);
+            out.write((int) b);
         } catch (final IOException e) {
             throw new RuntimeIOException(e);
         }
     }
 
+    @Override
+    public String toString() {
+        return "DefaultBitOutputStream: "
+                + Integer.toBinaryString(bufferByte).substring(0, bufferedNumberOfBits);
+    }
+
+    private void write_int_LSB_0(final int value, final int nofBitsToWrite) {
```

Comment:
Could you add a comment to this method, it's not obvious from the name what's happening.

Reply:
Should this be renamed in terms of endian-ness?